Configuration/Eras/python/Era_Run3_UPC_2023_cff.py

```diff
@@ -0,0 +1,9 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+from Configuration.ProcessModifiers.egamma_lowPt_exclusive_cff import egamma_lowPt_exclusive
+from Configuration.Eras.Modifier_highBetaStar_2018_cff import highBetaStar_2018
+from Configuration.Eras.Modifier_run3_upc_cff import run3_upc
+from Configuration.Eras.Modifier_run3_egamma_2023_cff import run3_egamma_2023
+
+Run3_UPC_2023 = cms.ModifierChain(Run3, egamma_lowPt_exclusive, highBetaStar_2018, run3_upc, run3_egamma_2023)
```

Comment:
`Run3_2023_UPC` would follow better the existing naming convention. How about using `Run3_2023` as the starting point?

Reply:
I implemented your suggestion